Target-MVD-Mathias/Manager/UserDataManager.swift

```diff
@@ -39,6 +39,10 @@ class UserDataManager: NSObject {
     return self.getUserObject()?.id ?? 0
   }
   
+  class func isUserFromFacebook() -> Bool {
```

Comment:
Maybe just a simple key for this that you can set inside the login with Facebook method and thats it. WDYT?

Reply:
Yeah, I think that approach is better, I'll change it